clusters/JoinBuildFarm.md

```
@@ -19,7 +19,7 @@ Make a softlink to each folder.
 ```
 $ cd build-clusters/vsphere
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)